### PR TITLE
WIP: install the tile server in an LXC container

### PR DESCRIPTION
**Pull request by yohanboniface**

### CODE REVIEW

File: remote/host/__main__.py [hunk 1]
```diff
@@ -78,4 +97,4 @@ def letsencrypt():
 
 
 if __name__ == '__main__':
-    main('pianoforteqa')
+    main('root@51.15.239.29')
```
Comment (kent1D) [inline remote/host/__main__.py]: Wouldn't it be better to have local configuration somewhere for IPs ?

File: remote/config.yml [hunk 2]
```diff
@@ -9,6 +9,12 @@ domains:
 - c.tiles.quaidorsay.fr
 ssl: true
 psql_version: 10
+tilery:
+  download_url: https://download.geofabrik.de/africa/burundi-latest.osm.pbf
+  replication_url: https://download.geofabrik.de/africa/burundi-updates/
+  proxy_command: ssh -q root@51.15.239.29 -W 10.10.10.10:22
```
Comment (kent1D) [inline remote/config.yml]: Here also for the IP

File: remote/host/__main__.py (hunk 1 above)
Comment (yohanboniface) [reply remote/host/__main__.py]: it's WIP :)

Comment (kent1D) [reply remote/host/__main__.py]: I know I know, it's just to prove that I don't sleep :p

**Pull request merged**